bin/pycbc_single_template

```diff
@@ -259,12 +259,12 @@ with ctx:
         row = WaveformArray.from_kwargs(
             mass1=inj.mass1,
             mass2=inj.mass2,
-            spin1x=inj.spin1x,
-            spin1y=inj.spin1y,
-            spin1z=inj.spin1z,
-            spin2x=inj.spin2x,
-            spin2y=inj.spin2y,
-            spin2z=inj.spin2z)
+            spin1x=inj.spin1x if hasattr(inj, 'spin1x') else 0,
+            spin1y=inj.spin1y if hasattr(inj, 'spin1y') else 0,
+            spin1z=inj.spin1z if hasattr(inj, 'spin1x') else 0,
```

Comment:
Typo?

Reply:
@titodalcanton Arggh! Indeed you are right.